script.js

```
@@ -0,0 +1,150 @@
+      }
+      this._timeOut(this.#timeOut);
+      this._clear();
+      this._displayspinner();
+      const res = await fetch("https://api.quotable.io/random");
+      const data = await res.json();
+      this.#currentQuote = data.content;
+      this.#currentAuthor = data.author;
+      this._displayNewQuote(this.#currentQuote, this.#currentAuthor);
+      if (data.length > 200) throw new Error("too long quote");
+    } catch (err) {
+      throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)